docs/src/npzd_model_benchmark.md

```diff
@@ -0,0 +1,208 @@
+# [Benchmark: Solution of an NPZD model](@id benchmark-npzd)
+
+We use the NPZD model [`prob_pds_npzd`](@ref) to assess the efficiency of different solvers.
+
+Standard methods have difficulties to solve this problem accurately, at least for low tolerances.
+
+```@example NPZD
+using OrdinaryDiffEq, PositiveIntegrators
+using Plots
+
+# select problem
+prob = prob_pds_npzd
+
+# compute reference solution (standard tolerances are too low)
```

Comment:
Do you mean not strict enough? The question is if you consider `1e-5` to be lower (smaller) than `1e-13`